Jenkinsfile

```diff
@@ -1072,17 +1075,20 @@ pipeline {
                     }
                     post {
                         always {
-                            junit testResults: 'bundles/junit-report-*.xml', allowEmptyResults: true
+                            junit testResults: 'bundles/junit-report-*.xml', allowEmptyResults: false
```

Comment:
IIRC, I used the "allow empty" because this step is in the "post" step, so if we failed the actual tests, we don't really care what happens next, just try to upload results if possible, and run the cleanup after

Reply:
Ok. Can change back